src/libs/tmdk/src/lib/component-type.ts

```diff
@@ -106,8 +109,15 @@ async function waitForComponentTypeActive(
   workspaceId: string,
   componentTypeId: string
 ) {
+  let timeout = 50;
```

Comment:
50 seems like a long timeout, what is the usual use case for this method? You can also make the timeout a parameter

Reply:
I agree, I added an arbitrary timeout in case an error happens because there was a FIXME there, completely reasonable to lower it. I have never encountered a case when this timeout would be necessary, it's mostly a failsafe against an infinite loop.

The use case of the method in general is to give a time buffer after creating a component type because if you notice in the console, during creation we are in "creating" or "updating" state, and we need to wait for that to become "active" before proceeding.

Reply:
Got it

Reply:
Got it